cypress/README.md

```diff
@@ -4,23 +4,23 @@
 
 - OSX Ventura with Git and Brew
 - Linux with bash curl and git
-- Windows with NodeJS 18 (untested)
+- Windows with NodeJS 16 (untested)
 - Sauce Labs Account
 
 ## Run the demo
 
-- Install Node.js 18 on Mac:
+- Install Node.js 16 on Mac:
 
 ```sh { name=nodejs-mac }
-brew install node@18
+brew install node@16
 ```
 
-- Install Node.js 18 + Dependencies on Linux:
+- Install Node.js 16 + Dependencies on Linux:
 
 ```sh { name=nodejs-linux }
 curl -fsSLO https://deb.nodesource.com/nsolid_setup_deb.sh
 chmod +x nsolid_setup_deb.sh
-./nsolid_setup_deb.sh 18
+./nsolid_setup_deb.sh 16
```

Comment:
hey - this url will show a deprecation warning and stop scrip execution for a while. It's not recommended by nodesource to use it and they're doing everything you keep updating node. Instead I really recommend showing nvm instructions instead. What do you think?